MREUnityRuntime/MREUnityRuntimeLib/Core/Actor.cs

```diff
@@ -75,7 +75,12 @@ public MWScaledTransform LocalTransform
         {
             get
             {
-                _localTransform = _localTransform ?? transform.ToLocalTransform();
+                if (_localTransform == null)
+                {
+                    _localTransform = new MWScaledTransform();
+                    _localTransform.ToLocalTransform(transform);
+                }
```

Comment:
Looks like Tom's changes snuck in. Can you rebase your changes to 0.12.n?

Reply:
Re-activating this thread. Tom's code still present.

---
In reply to: [307063794](https://github.com/microsoft/mixed-reality-extension-unity/pull/92#discussion_r307063794) [](ancestors = 307063794)